src/mbgl/tile/tile.hpp

```diff
@@ -25,8 +27,22 @@ namespace style {
 class Layer;
 } // namespace style
 
+using TilePoint = Point<int16_t>;
+
 class Tile : private util::noncopyable {
 public:
+    static TilePoint pointFromTileCoordinate(const UnwrappedTileID& tileID, const TileCoordinatePoint& p) {
```

Comment:
This is more closely related to to `TileCoordinate` than `Tile` -- let's move it there.